### PR TITLE
Support NPU for SSD MobileNetV1 example

### PR DESCRIPTION
**Pull request by huningxin**
This change converts the float32 weigths to float16 at loading time. And it uses WebNN cast operator to convert float32 inputs to float16 before compute and convert float16 outputs back to float32 after compute.

/cc @mingmingtasd @ibelem

### CODE REVIEW

File: common/utils.js [hunk 1]
```diff
@@ -34,7 +34,57 @@ export async function getBufferFromUrl(url) {
   return arrayBuffer;
 }
 
-export async function buildConstantByNpy(builder, url) {
+// ref: http://stackoverflow.com/questions/32633585/how-do-you-convert-to-half-floats-in-javascript
+export const toHalf = (function() {
```
Comment (Honry) [inline common/utils.js:38]: This method has been added in @mingmingtasd's PR https://github.com/webmachinelearning/webnn-samples/pull/226. Please help review and merge that one then we can ignore this change in your PR.

**Review by Honry — COMMENTED** [review]
@huningxin, thanks for the PR! Good idea to cast the float 32 to float 16 directly for NPU, maybe we can support fp16 in other samples like this if there's no precision issue.

File: common/utils.js (hunk 1 above)
Comment (huningxin) [reply common/utils.js:38]: SGTM, I'll rebase once #226 merged.

**Review by Honry — APPROVED** [review]
LGTM, thanks!

File: common/utils.js (hunk 1 above)
Comment (huningxin) [reply common/utils.js:38]: #226 may not need `toHalf` helper once it uses cast operator.

**Pull request merged**